packages/bun-types/bun.d.ts

```diff
@@ -5549,6 +5617,34 @@ declare module "bun" {
       maxBuffer?: number;
     }
 
+    interface SpawnSyncOptions<In extends Writable, Out extends Readable, Err extends Readable>
+      extends BaseOptions<In, Out, Err> {}
+
+    interface SpawnOptions<In extends Writable, Out extends Readable, Err extends Readable>
+      extends BaseOptions<In, Out, Err> {
+      /**
+       * If true, stdout and stderr pipes will not automatically start reading
+       * data. Reading will only begin when you access the `stdout` or `stderr`
+       * properties.
+       *
+       * This can improve performance when you don't need to read output
+       * immediately.
```

Comment:
_improve performance_ is kind of BS. If the user is optimizing for latency, then this makes performance worse.

I would be exact here and say _May improve data bandwidth._ or something similar since that's actually more likely what this means

Reply:
"Can" does some heavy lifting here, but can update this to be more correct